### PR TITLE
Another example, using with function

### PR DESCRIPTION
**Pull request by septyaman**
We can use a function that can be called at any time, and ease.

### CODE REVIEW

File: php/example_captha-with_function [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  // Register API keys at https://www.google.com/recaptcha/admin
+  define('G_Recaptcha_Publickey','Your_captcha_public_keys');
+  define('G_Recaptcha_Secretkey','Your_captcha_secret_keys');
+ 
+ 
+ /* ---- The function example ----- */
+ 
+ // ex Function to display the form
+ 
+  function display_my_recaptcha(){
```
Comment (septyaman) [inline php/example_captha-with_function]: This Function not don't required the captcha class. Only to display the form, but site key and lang id must defined.

File: php/example_captha-with_function [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  
+  // Register API keys at https://www.google.com/recaptcha/admin
+  define('G_Recaptcha_Publickey','Your_captcha_public_keys');
+  define('G_Recaptcha_Secretkey','Your_captcha_secret_keys');
+ 
+ 
+ /* ---- The function example ----- */
+ 
+ // ex Function to display the form
+
```
Comment (septyaman) [inline php/example_captha-with_function]: This Function not don't required the captcha class. Only to display the form, but site key and lang id must defined.

File: php/example_captha-with_function [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  function display_my_recaptcha(){
+
+	 $pub = ( defined('G_Recaptcha_Publickey') ? G_Recaptcha_Publickey : 0 );
+	 return $recaptchaform = '<div class="g-recaptcha" data-sitekey="'.$pub.'"></div>
+	 <script type="text/javascript" src="https://www.google.com/recaptcha/api.js?hl='.( defined('G_Recaptcha_Lang') ? G_Recaptcha_Lang : 'en' ).'">
+	 </script>';
+  }
+  
+  // ex Functions to controlling the response (true/false)
+
```
Comment (septyaman) [inline php/example_captha-with_function]: captcha class are required. Only to handel response from user, and check if valid or not

File: php/example_captha-with_function [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+ 
+ /* ---- Your Api Set Here ----- */
+
```
Comment (septyaman) [inline php/example_captha-with_function]: Here, Please set your API ID

File: php/example_captha-with_function [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  $reCaptcha = new ReCaptcha($secret);
+  
+  if ( $rem && $response ) $resp = $reCaptcha->verifyResponse($rem,$response);
+  
+  if ($resp != null && $resp->success) return true;
+  else return false; // exit;
+  
+ }
+ 
+
```
Comment (septyaman) [inline php/example_captha-with_function]: Now start your php application

**Comment by rowan-m** [comment]
Hopefully these issues should be resolved for you in the new v1.1. There's also a new contribution process and I'm watching the PRs, so if you would like to contribute then please feel free to submit again!

**Pull request closed without merging**